optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -1340,6 +1340,13 @@ class _OVModelForWhisper(OVModelForSpeechSeq2Seq, WhisperForConditionalGeneratio
     # force the use of the WhisperForConditionalGeneration generate and prepare_inputs_for_generation methods
     generate = WhisperForConditionalGeneration.generate
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Restore multilingual model transcribe task by default after model export
+        if is_transformers_version(">=", "4.50"):
+            if self.generation_config.is_multilingual and self.generation_config.forced_decoder_ids is None:
+                self.config.forced_decoder_ids = [[1, None], [2, self.generation_config.task_to_id["transcribe"]]]
```

Comment:
why do we need to set `self.config.forced_decoder_ids` in this specific case ?

Reply:
there is bug in transformers that does not allow serialize and read from disk generation config with forced_decoder_ids. As The result we lost this information. however Whisper generation pipeline uses forced_decoder_ids from generation_config and if it is not provided, it checks model.config for getting them. These ids forcing which task and language model will use by default if user does not specified that (None means language is not provided and allow autodetection and transcribe as task) that is matched with default behavior. 

We modify config and generation config during export for preserving this behavior, while if model export invoked using from_pretrained, there is no config reloading.

Do you think, it will be better to reload it instead of forcing?

Reply:
thanks a lot @eaidova, looks good to me !